README.md

```diff
@@ -500,6 +494,54 @@ it is always better to use less general tools whenever possible:
   (cc/quick-bench (valid? (range 10)))) ; Execution time mean : 0.12µs
 ```
 
+## Vector schemas
+
+You can use `:vector` to describe homogeneous Clojure vectors.
+
+```clojure
+(m/validate [:vector int?] [1 2 3])
+;; => true
+
+(m/validate [:vector int?] (list 1 2 3))
+;; => false
+```
+
+A `:tuple` schema describes a fixed length Clojure vector of heterogeneous elements:
+
+```clojure
+(m/validate [:tuple keyword? string? number?] [:bing "bang" 42])
+;; => true
+```
+
+To create a vector schema based on a seqex, use `:and`.
```

Comment:
What's a `seqex`?

Reply:
"Sequence Expressions", many names for the same thing. I'll add that.